pepper-apis/dsm-core/src/main/java/org/broadinstitute/lddp/email/SendGridClient.java

```diff
@@ -20,29 +20,27 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * Used for integrating with SendGrid...
+ * SendGridClient is a legacy wrapper for using SendGrid, carried over from
+ * the legacy DDP repo, and used exclusively by DSM.
  */
 public class SendGridClient {
 
     private static final Logger logger = LoggerFactory.getLogger(SendGridClient.class);
 
     private static final String LOG_PREFIX = "SendGrid";
+    private final SendGrid sendGrid;
 
-    private String sendGridKey;
-    private JsonObject settings = null;
-
-    public void configure(String sendGridKey, JsonObject settings) {
```

Comment:
Calls to `configure()` have been replaced by adding `settings` params to various methods.